test/test_basic_submission.py

```diff
@@ -146,6 +151,31 @@ def test_drs_workflow_in_terra(self):
         with self.subTest('Dockstore Workflow Run Completed Successfully'):
             self.assertEqual(status, "Done")
 
+    def test_pfb_handoff_from_gen3_to_terra(self):
+        time_stamp = str(datetime.datetime.now()).split('.')[0].replace(':', '').replace(' ', '_').replace('-', '_')
+        workspace_name = f'drs_test_{time_stamp}_delete_me'
+
+        with self.subTest('Create a terra workspace.'):
+            response = create_terra_workspace(workspace=workspace_name)
+            self.assertTrue('workspaceId' in response)
+            self.assertTrue(response['createdBy'] == 'biodata.integration.test.mule@gmail.com')
+
+        with self.subTest('Import static pfb into the terra workspace.'):
+            response = import_pfb(workspace=workspace_name)
+            self.assertTrue('jobId' in response)
+
+        with self.subTest('Check on the import static pfb job status.'):
+            response = pfb_job_status_in_terra(workspace=workspace_name, job_id=response['jobId'])
+            # this should take 5-10 seconds
+            while response['status'] == 'Translating':
+                time.sleep(2)
+                response = pfb_job_status_in_terra(workspace=workspace_name, job_id=response['jobId'])
+            self.assertTrue(response['status'] == 'ReadyForUpsert')
+
+        with self.subTest('Delete the terra workspace.'):
+            response = delete_terra_workspace(workspace=workspace_name)
+            self.assertTrue(b'will be deleted' in response)
```

Comment:
Consider asserting a status code instead, and possibly making subsequent requests until a 404 is returned.

Otherwise this subtest will give false negatives if the response verbiage changes.

Reply:
Makes sense to me.